packages/odie-client/src/components/send-message-input/resizable-textarea.tsx

```diff
@@ -22,10 +22,6 @@ export const ResizableTextarea: React.FC< {
 		? __( 'Just a moment…', __i18n_text_domain__ )
 		: __( 'Type a message…', __i18n_text_domain__ );
 
-	if ( shouldDisableInputField ) {
```

Comment:
Can we stick with the same naming of the properties? eg. Can we set instead of `shouldDisableInputField` to `submitDisabled`? that way we know its the getter and the setter by nomemclature.